nexus/test-utils/src/background.rs

```diff
@@ -71,19 +53,40 @@ pub async fn wait_background_task(
 }
 
 /// Given the name of a background task, activate it, then wait for it to
-/// complete. Return the last polled `BackgroundTask` object.
+/// complete. Return the `BackgroundTask` object from this invocation.
 pub async fn activate_background_task(
     internal_client: &ClientTestContext,
     task_name: &str,
 ) -> BackgroundTask {
-    let task = NexusRequest::object_get(
-        internal_client,
-        &format!("/bgtasks/view/{task_name}"),
-    )
-    .execute_and_parse_unwrap::<BackgroundTask>()
-    .await;
+    // If it is running, wait for an existing task to complete - this function
```

Comment:
Is this change actually necessary to fix this flake?  If not, can we omit it from this PR (or merge it in a separate PR?)

Reply:
I don't think this change is necessary for this particular flake, I can pull it out